src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/SpgAatfAeReturnDataCsvDataTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿namespace EA.Weee.DataAccess.Tests.DataAccess.StoredProcedure
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using EA.Weee.Tests.Core;
+    using EA.Weee.Tests.Core.Model;
+    using Xunit;
+
+    public class SpgAatfAeReturnDataCsvDataTests
```

Comment:
Can we add additional tests, i.e. to check the filters work correctly

Reply:
Done